lib/blockscore/error/blockscore_error.rb

```diff
@@ -1,27 +0,0 @@
-module BlockScore
-  class BlockscoreError < StandardError
```

Comment:
this should probably just be `BlockScore::Error` since we already have the namespace